pygameMenu/widgets/textinput.py

```diff
@@ -524,14 +525,24 @@ def _move_cursor_right(self):
         self._cursor_position = min(self._cursor_position + 1, len(self._input_string))
         self._update_renderbox(right=1)
 
+    def _blur(self):
```

Comment:
Instead of create new privates methods, why don't you override the `set_selected()` method?
For instance, in TextInput class:

```
    def set_selected(self, selected=True):
        super(TextInput, self).set_selected(selected)
        if not selected:
            self._key_is_pressed = False
            self._mouse_is_pressed = False
            self._keyrepeat_mouse_ms = 0
            self._cursor_render = True
        else:
            ....
```

Reply:
I created blur/focus keeping in mind the same behaviour of HTML inputs.Just implementing focus and blur to a specific widget is clearer for me, maybe I'm wrong? 🥴

Reply:
That's ok for the reason, I'm not very familiar with HTML development.